provider/resource_keycloak_openid_client_authorization_aggregate_policy.go

```diff
@@ -0,0 +1,185 @@
+package provider
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func resourceKeycloakOpenidClientAuthorizationAggregatePolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakOpenidClientAuthorizationAggregatePolicyCreate,
+		Read:   resourceKeycloakOpenidClientAuthorizationAggregatePolicyRead,
+		Delete: resourceKeycloakOpenidClientAuthorizationAggregatePolicyDelete,
+		Update: resourceKeycloakOpenidClientAuthorizationAggregatePolicyUpdate,
+		Importer: &schema.ResourceImporter{
+			State: resourceKeycloakOpenidClientAuthorizationAggregatePolicyImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"resource_server_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"realm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"decision_strategy": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"owner": {
```

Comment:
I'm a bit confused on what the `owner`, `resources`, and `scopes` attributes are supposed to represent.  I see these attributes exist for all of the policy resources but I can't seem to find them in the Keycloak GUI, nor do I see these being used in the `client_authorization_policies.tf` file that you added.  Could you help me understand why these are needed?

Reply:
They're not! I'll remove them.

Reply:
Sorry, I should have been more clear with this.  I saw these attributes on all of the policy resources you added, but I only left the comment on this file.  Could you take a look at all of the other policy resources and see if these attributes are needed or used?  If not, could you remove them and also remove them from the corresponding struct in the `keycloak` package?

Reply:
No worries! I removed all the unnecessary attributes.